AppleMusicClone/Views/SearchViewModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SearchListViewModel.swift
+//  AppleMusicClone
+//
+//  Created by 최동권 on 2022/07/08.
+//
+
+import Foundation
+import MusicKit
+
+class SearchViewModel: ObservableObject {
+    @Published var recentlySearched: [Album]?
+    @Published var recentlySearchedAlbum: [Message] = []
+  
+    func fetchRecentlySearchedList() {
+        let collectionPath = "albums"
+        FirebaseManager.shared.firestore.collection(collectionPath).getDocuments { querySnapshot, err in
+            guard let documents = querySnapshot?.documents else { return }
+                  for document in documents {
+                      guard let element = try? document.data(as: Message.self) else { return }
+                      self.recentlySearchedAlbum.append(element)
+                  }
+            let set = Set(self.recentlySearchedAlbum)
```

Comment:
```suggestion
            let set = Set(self.recentlySearchedAlbum.sorted { $0.sentDate > $1.sentDate })
```

제가 위 SearchListView파일에서 말씀드린 의도는 뷰파일에서의 sort보다 뷰모델에서 로직을 구현하자라는 의미였습니다!

Reply:
`3025651326714b180a79aad6d3abccb6ba3a2c49` 에서 반영 완료